internal/service/eks/pod_identity_association.go

```diff
@@ -30,42 +30,43 @@ import (
 	"github.com/hashicorp/terraform-provider-aws/names"
 )
 
-// Function annotations are used for resource registration to the Provider. DO NOT EDIT.
 // @FrameworkResource(name="Pod Identity Association")
 // @Tags(identifierAttribute="association_arn")
-func newResourcePodIdentityAssociation(_ context.Context) (resource.ResourceWithConfigure, error) {
-	r := &resourcePodIdentityAssociation{}
+func newPodIdentityAssociationResource(_ context.Context) (resource.ResourceWithConfigure, error) {
```

Comment:
@ewbankkit these were generated by `skaff` - are there updates coming for it to use the recommended conventions?

Reply:
My personal taste 😄.